styles.css

```diff
@@ -0,0 +1,232 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.first-card {
+  background-color: #e5e5e5;
+  height: 100vh;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.first-card-section {
+  margin-bottom: 90px;
+}
```

Comment:
- [ ] Good job 👏  on following the original template design however you still need some last improvements on your design. Make your card projects have a fixed width (`width: 327px`) and add a border around the card just like the original design is(`border: 1px solid #DFE1E6`).